extensions/ql-vscode/package.json

```diff
@@ -1222,7 +1222,7 @@
     }
   },
   "lint-staged": {
-    "./**/*.{json,css,scss,md}": [
+    "./**/*.{json,css,scss}": [
```

Comment:
Does anyone feel strongly about us not "prettifying" markdown files? 🙏🏽 

I mainly removed this to stop my test output from getting overwritten, and this was easier than excluding specific folders...